testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -537,6 +537,34 @@ protected void checkInformationSchemaViewsForMaterializedView(String schemaName,
                 .containsAll("VALUES '" + viewName + "'");
     }
 
+    @Test
+    public void testMaterializedAndNormalViewsTogether()
+    {
+        if (!hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW) || !hasBehavior(SUPPORTS_CREATE_VIEW)) {
+            throw new SkipException("Either materialized or normal views not supported");
+        }
+
+        String catalogName = getSession().getCatalog().orElseThrow();
+        String schemaName = getSession().getSchema().orElseThrow();
+
+        String normalViewName = "test_views_together_normal_" + randomTableSuffix();
```

Comment:
just `viewName` is OK

(else normalView -> regularView)

Reply:
Went for `regularView` to be more explicit. Not much longer.